docs/apis-tools/camunda-api-rest/camunda-api-rest-data-fetching.md

```diff
@@ -0,0 +1,324 @@
+---
+id: camunda-api-rest-data-fetching
+title: "Data fetching"
+description: "Learn about fetching data using the Orchestration Cluster API."
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+Many Orchestration Cluster API resources like process definitions, user tasks, users, and tenants provide endpoints to fetch related data.
+
+Those resources can offer search endpoints to query by POST method and a given query request object where applicable. The structure of such [search requests](#search-requests) always follows the same schema and so do the [search responses](#search-responses), always returning a list of items matching the query criteria, accompanied by search result metadata.
+
+Resources can also support querying subordinate resources. For example, for users and groups, with group search being available at `POST /v2/groups/search`, a group's users can be retrieved using `POST /v2/groups/{groupId}/users/search`. Each resource determines independently if subordinate resources can be accessed this way.
+
+Search endpoints can also be used to access entity instances with a unique identifier directly. As an alternative, resources can also provide GET method endpoints for fetching the data of single instances. This is done by a specific key parameter in the URL path, for example `GET /v2/user-tasks/{userTaskKey}`.
+
+All those data retrieval endpoints offer near-real-time data consistency guarantees, providing runtime and historic data that has been processed by the [Camunda Exporter](/self-managed/zeebe-deployment/exporters/camunda-exporter.md`).
```

Comment:
```suggestion
All those data retrieval endpoints offer near-real-time data consistency guarantees, providing runtime and historic data that has been processed by the [Camunda Exporter](../../self-managed/zeebe-deployment/exporters/camunda-exporter.md`).
```

Reply:
That's interesting, absolute paths references should work just fine, they work for other links as well 🤔

Reply:
Yeah, I guess we'd have to define linting in the docs if we wanted to avoid either. Both have their pros and cons, so 🤷